apple-pie/index.html

```diff
@@ -4,36 +4,39 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="/apple-pie/styles/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="styles/style.css">
```
Acuerdate el index.html esta dentro de la propia carpeta de apple-pie, si lo pones esta intentando buscar dentro de apple-pie, la carpeta apple-pie!